pulley/src/disas.rs

```diff
@@ -274,7 +274,7 @@ impl<'a> OpVisitor for Disassembler<'a> {
                 write!(&mut self.disas, "{space}{byte:02x}").unwrap();
                 need_space = true;
             }
-            for _ in 0..11_usize.saturating_sub(size) {
+            for _ in 0..12_usize.saturating_sub(size) {
```

Comment:
This was done to make the disassembly a bit prettier since otherwise the hexdump and instructions might be side-by-side (no whitespace between)